resources/linux/debian/postinst.template

```diff
@@ -42,43 +42,55 @@ if [ "@@NAME@@" != "code-oss" ]; then
 		db_get @@NAME@@/add-microsoft-repo || true
 	fi
 
-	# Determine whether to install the repository source list
-	WRITE_SOURCE=0
-	if [ "$RET" = false ]; then
+	# Determine whether to write the Microsoft repository source list
+	WRITE_SOURCE='no'
+	if [ "$CODE_INSTALL_WITH_REPO" = false ]; then
```

Comment:
Should this instead be a string of undefined, 0 or 1? Can you have boolean environment variables?

Reply:
In comparisons, `= false` is equivalent to `= 'false'`, so I don't think the variable can be an actual boolean variable.

I'm thinking I can actually remove this environment variable for this PR considering I added an interactive terminal check below.